src/transformState.js

```diff
@@ -5,7 +5,21 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(state, action.extraData);
+        break;
+      case 'removeProperties':
+        action.keysToRemove.map(key => delete state[key]);
+        break;
+      case 'clear':
+        Object.keys(state).map(key => delete state[key]);
+        break;
+      default:
```

Comment:
Great Job👏. However, maybe it is better to make function to return something by default?